src/registry/utils/plugins/external-links.ts

```diff
@@ -47,12 +50,16 @@ const externalLinks = (options: Options = {}) => {
             node.properties.target = target || defaultTarget;
           }
 
-          if (className) {
-            node.properties.className =
-              (node.properties.className
-                ? `${node.properties.className} `
-                : '') + className;
-          }
+          const isUkrainian = isUrlUkrainian(url);
+
+          node.properties.className = classNames(
+            node.properties.className,
+            className,
+            {
+              'wd-foreign': !isUkrainian,
```

Comment:
Це дуже загальний клас, давай зробимо його більш специфічним
```suggestion
              'wd-link__foreign': !isUkrainian,
```